packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -88,6 +90,59 @@ describe('ui:create:atomic', () => {
     page = await openNewPage(browser, page);
   });
 
+  function projectFileExist(path: string) {
+    expect(
+      existsSync(join(getProjectPath(projectName), ...path.split('/')))
+    ).toBe(true);
+  }
+
+  function projectFileDoesNotExist(path: string) {
+    expect(
+      existsSync(join(getProjectPath(projectName), ...path.split('/')))
+    ).toBe(false);
+  }
+
+  it('should create the proper template files', () => {
+    const createdFilesPaths = [
+      'package.json',
+      'package-lock.json',
+      '.gitignore',
+      '.env',
+      '.env.example',
+      'README.md',
+      'start-netlify.mjs',
+      'netlify.toml',
+      'tsconfig.json',
+      'stencil.config.ts',
+      'lambda',
+      'src/index.ts',
```

Comment:
Could be an array, but the use of `/` is easier to read IMO and keep in sync with the paths.